src/aliases/api/types/alias.go

```diff
@@ -0,0 +1,61 @@
+package types
+
+import aliasent "github.com/consensys/quorum-key-manager/src/aliases/entities"
+
+type Alias struct {
+	Key   AliasKey   `json:"key"`
+	Value AliasValue `json:"value"`
+
+	registryName RegistryName
+}
+
+func FromEntityAlias(ent aliasent.Alias) Alias {
```

Comment:
In order to be more consistant we should have `formatters` folder to encapsulate Request/Response from/to Entities logic

Reply:
Why can't we keep it in the types package? It's type transformation from/to domain entities.
Domain entities shouldn't know about other format, but the other way around is good. Therefore, I don't see the point of splitting even more packages when the subject of those formatters would be those types. And those formatters without those types would have no purpose at all.

Reply:
I don't feel the name `FormatXxx` does convey as much information as `FromXxx` and `ToXxx`.
In `stores/api/formatters`, some `FormatXxx` means it will transform to `Xxx`, and in other cases, `FormatXxx` means we transform `Xxx` into something else.
With From and To, we transform `From` a type from another package to the current one, and `To` would transform a current package type to another one. I think the name conveys directly more meaning without needing to read the whole signature and return types.

Reply:
I'll keep this suggestion in our future discussion, I'll rename like in `stores/api/formatters` for now.

Reply:
done